### DISCUSSION TITLE
Commercial License

### DISCUSSION THREAD

**Discussion by anveshr1** [OpenTalker/SadTalker#500] [Q&A]
The repository has MIT license which allows commercial usage while there is a disclaimer that restricts commercial use of the repo. Could you elaborate on the licensing for commercial purpose.

**Comment by vinthony** [comment]
Hi, we are working on with our company to grant the New Offical License for better usage, please wait for our update.

**Reply by anveshr1** [reply]
Is there any timeline that you are targeting for? Also, is commercial usage not allowed currently?
